regression/snapshot-harness/static-array-float/test.desc

```diff
@@ -0,0 +1,13 @@
+FUTURE
```

Comment:
Please explain (after another `--` at the end of the file) what isn't supported here so far.

Reply:
Added.